config/names/nightmares.txt

```diff
@@ -0,0 +1,27 @@
+a dead skrell
+a dead unathi
+a dead tajaran
+a dead vulpkanin
+a dead slime
+a dead ipc
+a dead vox
+a dead plasmaman
+a dead grey
+a dead kidan
+a dead diona 
+a dead drask
+the malf ai core
```

Comment:
> using malf intentionally

Reply:
i am not changing that..people use malf ic too..

Reply:
use `rogue ai`.... not this stupid behavioral playerbase thing

Reply:
Malf AI is perfectly acceptable terminology, and it's been used longer than the "reee calling a gamemode by it's name!" trend.

Reply:
Finally enough, the name of the gamemode was malf AI, if I remember right.

Also, hi, everyone.